Getter implementation is not initialized

Hello again! I tried to `shipyard run github.com/shipyard-run/blueprints//consul-docker` and got the following panic.
```
Running configuration from:  github.com/shipyard-run/blueprints//consul-docker

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x19cd8d3]

goroutine 1 [running]:
github.com/shipyard-run/shipyard/pkg/clients.(*GetterImpl).Get(0xc00015a430, 0x7ffd524691a6, 0x31, 0xc0006cc0c0, 0x51, 0x1, 0x1)
        /home/runner/work/shipyard/shipyard/pkg/clients/getter.go:80 +0xc3
github.com/shipyard-run/shipyard/cmd.newRunCmdFunc.func1(0xc0001cf180, 0xc00015a4f0, 0x1, 0x1, 0x0, 0x0)
        /home/runner/work/shipyard/shipyard/cmd/run.go:93 +0x23ab
github.com/spf13/cobra.(*Command).execute(0xc0001cf180, 0xc00015a4c0, 0x1, 0x1, 0xc0001cf180, 0xc00015a4c0)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0x3806f60, 0x0, 0x1dd4f20, 0xc0000b4058)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/shipyard-run/shipyard/cmd.Execute(...)
        /home/runner/work/shipyard/shipyard/cmd/root.go:92
main.main()
        /home/runner/work/shipyard/shipyard/main.go:10 +0x52
```

```
shipyard version
Shipyard version: 0.0.33
```

--- a/pkg/shipyard/engine.go
+++ b/pkg/shipyard/engine.go
@@ -76,7 +76,7 @@ func GenerateClients(l hclog.Logger) (*Clients, error) {
 
 	nc := clients.NewNomad(hc, 1*time.Second, l)
 
-	bp := &clients.GetterImpl{}
+	bp := clients.NewGetter(false)
 
 	bc := &clients.SystemImpl{}
 

